src/openforms/prefill/api/urls.py

```diff
@@ -1,8 +1,29 @@
 from django.urls import path
 
-from .views import PluginAttributesListView, PluginListView
+from .views import (
+    PluginAttributesListView,
+    PluginListView,
+    PluginObjectsAPIAttributesListView,
+    PluginObjectsAPIObjecttypeListView,
+    PluginObjectsAPIObjecttypeVersionListView,
+)
 
 urlpatterns = [
+    path(
+        "plugins/objects-api/objecttypes",
+        PluginObjectsAPIObjecttypeListView.as_view(),
+        name="prefill-objects-api-objecttype-list",
+    ),
+    path(
+        "plugins/objects-api/objecttypes/<uuid:objects_api_objecttype_uuid>/versions",
+        PluginObjectsAPIObjecttypeVersionListView.as_view(),
+        name="prefill-objects-api-objecttype-version-list",
+    ),
+    path(
+        "plugins/objects-api/objecttypes/<uuid:objects_api_objecttype_uuid>/versions/<int:objects_api_objecttype_version>/attributes",
```

Comment:
```suggestion
        "plugins/objects-api/objecttypes/<uuid:objecttype_uuid>/versions/<int:version>/properties",
```
Property/properties is the terminology used in JSONSchema which is the standard behind objecttypes API.